src/streaming/controllers/PlaybackController.js

```diff
@@ -248,12 +248,14 @@ function PlaybackController() {
     function getStreamStartTime(ignoreStartOffset) {
         let presentationStartTime;
         let fragData = URIQueryAndFragmentModel(context).getInstance().getURIFragmentData();
-        let fragS = parseInt(fragData.s, 10);
-        let fragT = parseInt(fragData.t, 10);
-        let startTimeOffset = NaN;
+        let startTimeOffset = 0;
```

Comment:
In my case, when loading a parsed manifest object, if startTimeOffset is NaN, I will get the following error: Uncaught TypeError: Failed to set the 'currentTime' property on 'HTMLMediaElement': The provided double value is non-finite.

It needs to be 0 so that the stream starts from the beginning. I guess a better approach would be to leave the initial value as NaN, and if fragData is null, set startTimeOffset to 0?